index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="cont">
+        <h1>My Blog</h1>
+      </div>
+      <div class="container-one">
```

Comment:
Give a more semantic names for your html. Use Header Section, main, aside and footer. Class name should be descriptive, try a name that is understandable for someone reviewing your code.